appserver/security/core-ee/src/test/java/com/sun/enterprise/security/permissionsxml/GlobalPolicyUtilTest.java

```diff
@@ -37,201 +37,167 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
+// Portions Copyright [2018] [Payara Foundation and/or its affiliates]
+package com.sun.enterprise.security.permissionsxml;
 
-package com.sun.enterprise.security.perms;
-
-import java.security.Permission;
 import java.io.FilePermission;
+import java.net.URL;
+import java.security.Permission;
 import java.security.PermissionCollection;
 import java.util.Enumeration;
-import java.net.URL;
-
-import junit.framework.Assert;
 
-import org.junit.Test;
-import org.junit.AfterClass;
+import org.junit.Assert;
 import org.junit.BeforeClass;
+import org.junit.Test;
 
-//import com.sun.enterprise.security.perms.SMGlobalPolicyUtil;
-
-public class SMGlobalPolicyUtilTest {
+public class GlobalPolicyUtilTest {
 
     private static final String plfile = "server.policy";
-    
+
     @BeforeClass
     public static void setUpBeforeClass() throws Exception {
-        URL serverPF = SMGlobalPolicyUtilTest.class.getResource(plfile);
+        URL serverPF = GlobalPolicyUtilTest.class.getResource(plfile);
         System.out.println("policy file url = " + serverPF + ", path = " + serverPF.getPath());
-        System.setProperty(SMGlobalPolicyUtil.SYS_PROP_JAVA_SEC_POLICY, serverPF.getPath());
-    }
-
-    @AfterClass
-    public static void tearDownAfterClass() throws Exception {
-        
+        System.setProperty(GlobalPolicyUtil.SYS_PROP_JAVA_SEC_POLICY, serverPF.getPath());
     }
 
-    
     @Test
     public void testSystemPolicyPath() {
-        
-        System.out.println("path= " + SMGlobalPolicyUtil.domainCfgFolder);
-       
-        Assert.assertNotNull(SMGlobalPolicyUtil.domainCfgFolder);
+        System.out.println("path= " + GlobalPolicyUtil.domainCfgFolder);
+
+        Assert.assertNotNull(GlobalPolicyUtil.domainCfgFolder);
     }
-    
-    @Test    
+
+    @Test
     public void testTYpeConvert() {
-        
-        SMGlobalPolicyUtil.CommponentType t = SMGlobalPolicyUtil.convertComponentType("ejb");
+        CommponentType t = GlobalPolicyUtil.convertComponentType("ejb");
         System.out.println("Converted type = " + t);
-        Assert.assertEquals("Converted type should be Ejb", SMGlobalPolicyUtil.CommponentType.ejb, t);
-        
-        t = SMGlobalPolicyUtil.convertComponentType("ear");
+        Assert.assertEquals("Converted type should be Ejb", CommponentType.ejb, t);
```

Comment:
repetitive code could use a loop and componentType.values()

Reply:
👍